src/components/Banner/Banner.js

```diff
@@ -1,11 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
+import classnames from "classnames";
 
 import "./_banner.scss";
 
-export default function Banner({ isActive, children, type }) {
+export default function Banner({ isActive = true, children, type }) {
```

Comment:
Shouldn't the default state of this be `false`?

Reply:
I'd expect anyone using it to pass it that prop to toggle display but if not, it'll display by default.